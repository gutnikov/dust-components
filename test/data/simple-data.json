{
  "text": "This is a button too"
}