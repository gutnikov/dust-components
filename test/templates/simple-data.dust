<div>
	{@Button className="simple-button" text="{text}"/}
</div>
