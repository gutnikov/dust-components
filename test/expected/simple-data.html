<div>
    <button class="button simple-button">This is a button too</button>
</div>
